{{cookiecutter.project_slug}}/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,46 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.112.0/containers/docker-existing-docker-compose
+// If you want to run as a non-root user in the container, see .devcontainer/docker-compose.yml.
+{
+	"name": "Existing Docker Compose (Extend)",
+
+	// Update the 'dockerComposeFile' list if you have more compose files or use different names.
+	// The .devcontainer/docker-compose.yml file contains any overrides you need/want to make.
+	"dockerComposeFile": [
+		"../docker-compose.yml",
+		"docker-compose.yml"
```

Comment:
@TobiHobi is the `"docker-compose.yml"` needed? makes the build ptocess fail at my machine, because it looks for `.devcontainer/docker-compose.yml` and fails because it's not found

Reply:
```suggestion
	//	"docker-compose.yml"
```